compose-builder/upload-api-gateway-cert.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+# usage function for uploading tls cert:
+usage()
+{
+  echo "Usage: make upload-tls-cert CERT_INPUT_FILE=<full-path-for-cert-input-file> KEY_INPUT_FILE=<full-path-for-key-input-file> [EXTRA-SNIS=<comma-separated-list-for-server-name>]"
```

Comment:
Since this is the script itself change `make upload-tls-cert` to `upload-api-gateway-cert.sh ` and add description on running from make.

Reply:
changed